error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1899,4 +1900,19 @@ Duration after(StepVerifier.LastStep step, Duration duration) {
       return step.verifyTimeout(duration);
     }
   }
+
+  /**
+   * Don't unnecessarily convert {@link Collection} to {@link Stream} before creating {@link Flux}.
+   */
+  static final class FluxFromIterable<T> {
```

Comment:
I think a bit more up in the file makes sense, now they are below the `StepVerifier` rules. There is no strict pre-determined order yet, so right now I put it under the the "last" `iterable` related rule in the file.